users/vieting/experiments/switchboard/hybrid/feat/experiments.py

```diff
@@ -180,6 +180,69 @@ def run_baseline_mel():
         # noinspection PyUnresolvedReferences
         train_job.rqmt.update({"gpu_mem": 24, "mem": 10})
 
+def run_specaug_mel():
+    gs.ALIAS_AND_OUTPUT_SUBDIR = "experiments/switchboard/hybrid/feat/"
+
+    log_mel_args_8khz = {
+        "class": "LogMelNetwork", "wavenorm": True, "frame_size": 200, "frame_shift": 80, "fft_size": 256
+    }
+    nn_args = get_nn_args_baseline(
+        nn_base_args={
+            "lm80_fft256": dict(
+                returnn_args=dict(
+                    batch_size=14000,
+                    enable_specaug=True,
```

Comment:
`enable_specaug=True` is the default, so it can be remove. See all occurrences below as well.